src/index.js

```diff
@@ -145,15 +145,6 @@ function _renderToString(vnode, context, isSvgMode, selectValue, parent) {
 			rendered =
 				rendered +
 				_renderToString(child, context, isSvgMode, selectValue, parent);
-
-			if (
```

Comment:
We fixed this in 10.13.0, I honestly think we shouldn't put a preact/debug fix in our hot path

Reply:
I'm a bit out of the loop. Which issue in `preact/debug` did this section work around?

Reply:
This one https://github.com/preactjs/preact-render-to-string/pull/257